seml/add.py

```diff
@@ -136,7 +137,7 @@ def add_experiments(db_collection_name, config_file, force_duplicates, no_hash=F
     if 'conda_environment' not in seml_config:
         seml_config['conda_environment'] = os.environ.get('CONDA_DEFAULT_ENV')
 
-    # Set Slurm config with default parameters as fall-back option
+    # Set Slurm config with default parameters as fall-back option  
     slurm_config = merge_dicts(SETTINGS.SLURM_DEFAULT, slurm_config)
```

Comment:
On a related note: can we also make it possible to overwrite slurm parameters when starting experiments? Sometimes I screw up the partition or so and then I have to re-queue everything.

Reply:
That sounds like a good idea. However, I think we should open a follow-up issue on that and have a separate PR since the logic will be distinct from this one.

Reply:
Wouldn't this use the same interface? e.g. `--with slurm.sbatch_options.mem=25G` or so?

Reply:
I would use a separate argument and unify the style with `seml jupyter` where you have the `-sb` option to supply sbatch parameters. However, there you need to provide a dictionary which is not very user-friendly. In conjunction with this PR it could look something like:
```
seml <collection> add <yaml> -o dataset=imagenet -sb mem=25G
```
We could reuse the key-value parsing we introduce with this PR for the sbatch parameters but integrating both into one argument makes parsing more difficult and user always have to nest their parameters and sbatch options with `sbatch.xyz` and `config.xyz`.

Reply:
I agree that nesting with `config.` each time is not super user friendly. I suggest we do `-sb mem=25G partition=gpu_all` etc. separately (which is I think what you suggested)?

Reply:
Yes :)